contents.json

```diff
@@ -13076,6 +13076,16 @@
       "suggested_by": "@dkhamsing",
       "stars": 36
     },
+    {
+      "title": "Awesome ML",
+      "category-ids": ["developer"],
+      "description": "Discover, download, compile & launch different image processing CoreML models on iOS.",
+      "source": "https://github.com/eugenebokhan/Awesome-ML",
+      "screenshots": ["https://github.com/eugenebokhan/Awesome-ML/raw/master/Media/App-Demo-Transparent-Background.gif"],
+      "date_added": "April 2 2018",
+      "suggested_by": "@eugenebokhan"
```

Comment:
Looks like we missed comma at the end here `"suggested_by": "@eugenebokhan"`, could you add the one?